.gitignore

```diff
@@ -33,4 +33,3 @@
 /node_modules
 yarn-debug.log*
```

Comment:
actually, all of these should go in your global gitignore!
here's a good reference on what industry standards look like for these:
https://github.com/github/gitignore/blob/master/Node.gitignore

Reply:
done